src/main.ts

```diff
@@ -14,6 +14,8 @@ export default class ShikiPlugin extends Plugin {
 	settings!: Settings;
 	loadedSettings!: Settings;
 	updateCm6Plugin!: () => Promise<void>;
+	theme!: string;
```

Comment:
I am not the biggest fan of storing another reference to the active theme. I think a better approach would be to have a `getTheme` function like the following:
```ts
getTheme() {
    if (/* obsidian in light theme */) {
        return this.loadedSettings.lightTheme;
    } else {
        return this.loadedSettings.darkTheme;
    }
}
```